commit 177538eb191c297ac146c843ff68d80ecfe9a87b
Author: Natanael Copa <ncopa@alpinelinux.org>
Date:   Fri Jul 30 07:51:45 2010 +0000

    nptl: do not define _POSIX_SPAWN since its not implemented
    
    Building things like vlc checks if _POSIX_SPAWN is defined. Since
    posix_spawn is not implemented we dont define it.

diff --git a/libpthread/nptl/sysdeps/unix/sysv/linux/bits/posix_opt.h b/libpthread/nptl/sysdeps/unix/sysv/linux/bits/posix_opt.h
index 2550355..6fbdbb7 100644
--- a/libpthread/nptl/sysdeps/unix/sysv/linux/bits/posix_opt.h
+++ b/libpthread/nptl/sysdeps/unix/sysv/linux/bits/posix_opt.h
@@ -143,8 +143,10 @@
 /* We support spinlocks.  */
 #define _POSIX_SPIN_LOCKS	200809L
 
+#if 0
 /* The `spawn' function family is supported.  */
 #define _POSIX_SPAWN	200809L
+#endif
 
 /* We have POSIX timers.  */
 #define _POSIX_TIMERS	200809L
